src/modules/commands/discord/hasbrulefttheserver.ts

```diff
@@ -0,0 +1,25 @@
+import discord from 'discord.js'
+import { constants } from '@app/global/constants'
+import { Command } from '@commands/CommandHandler'
+
+export default new Command<discord.Message>({
+  keyword: 'hasbrulefttheserver',
+  description: 'check if Bru is online',
+  help: 'Usage: `{prefix}isbruonline` - checks if Bru is online',
```

Comment:
Wrong description and help. Also, this command should probably be hidden from the command list to prevent additional clutter.